### ISSUE TITLE
NG Classes Limited to NG+

### ISSUE THREAD

**Issue by Geardan** [larsiusprime/tdrpg-bugs#1055]

**Comment by Geardan** [comment]
I just encountered a problem, where I can't use any classes in NG that haven't yet been unlocked in NG+.

Steam DQ1 2.0.0, Win 10 64bit.

1. Finished NG w/ Heroic Mode, Deluxe Ver. (idk if either matter).
2. Started NG+, but didn't play first mission.
3. Went back to original game, ALL classes show 0/1 availability on every map I tried. I reloaded, exited game, etc. but no help.
4. After playing first mission in NG+, Slak was available in NG.
Sure enough, my class choices in NG maps are ALL limited to what is available at the point of NG+ I'm in. I dont' think that's intended behavior, but I guess I don't know for sure.
Actually, re-reading the OP, I think this is maybe a new/different bug. I'll list it as such.

New to GitHub, so sorry if I haven't given enough data, or if this is a duplicate issue (I didnt' see one like it). I'll try to provide any further answers to questions if you have them.

Love the game; just bought sequel today. Keep up the great work!
